docs/deployment/universal-profile.md

```diff
@@ -83,17 +83,17 @@ await lspFactory.UniversalProfile.deploy({
 };
 ```
 
-The following two will download the JSON file before hashing it and generating the proper [JSONURL](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md#JSONURL) value.
+The following two examples will download the JSON file before hashing it and generating the proper [JSONURL](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md#JSONURL) value.
```

Comment:
Suggesting to remove ~~before hashing it and~~ and replace by:
(but not sure if it is still correct)

```markdown
The following two examples will download the JSON file and generate the proper [JSONURL](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md#JSONURL) value.
```